docs/configuration/configuration.md

```diff
@@ -473,6 +473,30 @@ authorization:
   # It is mutually exclusive with `credentials`.
   [ credentials_file: <filename> ]
 
+# Optionally configures OAuth 2.0 authentication using the
```

Comment:
Can we have that in a specific section instead of repeating eveywhere? this is a lot of parameters.